tests/framework/extensions/machinepools/machinepools.go

```diff
@@ -191,3 +196,89 @@ func ScaleMachinePoolNodes(client *rancher.Client, cluster *v1.SteveAPIObject, n
 
 	return scaledClusterResp, nil
 }
+
+// AddRKE2K3SCustomClusterNodes is a helper method that will add nodes to the custom RKE2/K3S custom cluster.
+func AddRKE2K3SCustomClusterNodes(client *rancher.Client, cluster *v1.SteveAPIObject, nodes []*nodes.Node, rolesPerNode []string) error {
+	clusterStatus := &apiv1.ClusterStatus{}
+	err := v1.ConvertToK8sType(cluster.Status, clusterStatus)
+	if err != nil {
+		return nil
```

Comment:
```suggestion
		return err
```

Should this error be ignored?